Daytime-Nighttime-Interactions/slack-channel-usage.rst

```diff
@@ -0,0 +1,192 @@
+The ultimate goal is to facilitate effective communications to summit personnel by concentrating information into specific places and minimizing distractions.
+It is recognized that all project personnel are capable of contributing useful input, however, unless the people are very close to the on-going tasks there exists a significant awareness gap.
+Therefore, a flow via which anyone on the project can participate has been developed that passes through the key personnel which can address the project member's concerns and/or questions. 
+Additionally, this limits the number of simultaneous lines of communication that summit personnel are expected to monitor during their shifts.
+
+To better communicate the expectations for each of the roles, and demonstrating the information flow, the list of required Slack channels have been categorized into Tiers. 
+We use the "summit" prefix for channels that are tightly coupled to summit operations.
+
+- **Tier 0:** Real-time information that the applicable roles are always monitoring and are expected to immediately react to (when on shift and interacting with summit systems). 
+  Responding to a message may result in a loss of sky time, which is acceptable. 
+  Communication in this area meant to be extremely curt. 
+  People providing support are encouraged to do so via Zoom; not via Slack.
+
+- **Tier 1:** Used for asynchronous information communication regarding the state of summit systems. 
+  These channels are telescope specific, do not require immediate response, and are passively monitored by on-shift observers. 
+  It is primarily used for communication between summit personnel and externally requested support.
+
+- **Tier 2:** General discussion channels whose information and/or contents are not time critical.
+  The shift managers monitor these channels to relay any pertinent information between external support and the summit teams.
+
+- **Tier 3:** Channels not directly associated with current summit activities. 
+  These are channels for collaborative work, which make up the high majority of Slack channels.
+
+
+.. figure:: ./_static/Slack-channel-diagram.png
+    :name: Slack-channel-diagram
+
+    The expected level of participation by summit personnel to observation-related slack channels.
+
+
+Individual Channel Descriptions and Guidelines
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This section provides additional details regarding the use of each channel.
+Note that all channels require active monitoring by everyone to ensure the conversation stays inline with the channel's expected content.
+The shift managers are the primary disciplinarians but observers and others should police too!
+
+#summit-announce
+----------------
+This channel is used to announce activities that may impact other summit users.
+Announcements on this nor any other channel does **NOT** grant permission from personnel on the ground regarding whether or not it is safe to perform the test.
```

Comment:
This phrase is a bit confusing. I think the "nor" is giving the impression of a double negative. My non-native speaker brain is trying to rephrase it like this:

> Announcements in this, or in any other channel, does **NOT** grant permission from personnel on the ground regarding whether or not it is safe to perform the test.

I am afraid non-native speakers will have a hard time understanding the meaning of this phrase. Maybe rephrase it?